internal/utils/utils.go

```diff
@@ -27,8 +28,14 @@ func BuildImagePath(userImage, defaultImage, userVersion, defaultVersion string)
 		userImage = defaultImage
 	}
 
+	var checkNumber = regexp.MustCompile(`^[1-9].*`).MatchString
+
 	if userVersion == "" {
 		userVersion = defaultVersion
+	} else if checkNumber(userVersion) {
+		fmt.Println("version must start v; attaching v to the tag...")
```

Comment:
I think we probably don't even need to log anything here, we can just take whatever the user gives us as long as it passes the checks I suggested.
```suggestion
```